### ISSUE TITLE
EK3 Failure to initialize with no GPS

### ISSUE THREAD

**Issue by Hwurzburg** [ArduPilot/ardupilot#19072]
EK3 will not initialize without an attached GPS unless all normal EK3_SRC1_x using GPS are set to none. @tridge says this is a bug due to postponed buffer allocation until GPS returns data. Should be fixed for 4.1 ASAP.

**Platform**
[X  ] All
[  ] AntennaTracker
[  ] Copter
[  ] Plane
[  ] Rover
[  ] Submarine

**Comment by Hwurzburg** [comment]
@priseborough @tridge @rmackay9 any idea when this will be addressed? or should I change the wiki to have NONE recommended as sources?...

**Comment by rmackay9** [comment]
@Hwurzburg, thanks for the reminder, we should definitely fix this..

**Comment by Ryanf55** [comment]
Does this affect all vehicels?

**Comment by andyp1per** [comment]
I don't think this is true anymore, think we can close. I fly vehicles without GPS and the EKF initializes just fine.
